### DISCUSSION TITLE
Timer delay is shorter than expected when repeatedly triggered by a program running at full speed

### DISCUSSION THREAD

**Discussion by wrh3c** [nicolacimmino/PLC-14500#2] [General]
I found that if the timer is repeatedly triggered (OUT7) by a program running at full speed, and not based on the user pushing a button or flipping a switch, the timer period can be much shorter than expected.  I believe this is because C4, the 470 uF capacitor, doesn't have enough time to fully discharge before the next trigger.  I replaced C4 with a 10 uF capacitor and RV1 with a 1 Meg ohm potentiometer, which gives a consistent timer period with a range from 1/40 of a second to 11 seconds.

(Secondary issue:  The Tindie description says the timer range is 0.5s to 30s, but the "rev b" schematic says it is 1s to 7s which is correct when using the kit parts since they match the schematic.)

**Comment by wrh3c** [comment]
Here is the program I was working on when I noticed this issue:

[WRHcountV3.zip](https://github.com/nicolacimmino/PLC-14500/files/10985173/WRHcountV3.zip)

**Comment by nicolacimmino** [comment]
Thanks! Good idea to increase RV1 in order to decrease C4. I'll try it on couple of protos I have here tonight and if no issues (I don't expect any TBH), I'll change the BOM

**Comment by nicolacimmino** [comment]
Works perfectily, I have updated the BOM and will start shipping with the new values as soon as I get more components.
